nvflare/job_config/fed_app_config.py

```diff
@@ -36,6 +36,10 @@ def add_executor(self, tasks: List[str], executor: Executor):
         if not isinstance(executor, Executor):
             raise RuntimeError(f"workflow must be type of Executor, but got {executor.__class__}")
 
+        for item in self.executors:
+            if item.tasks == tasks:
+                raise RuntimeError(f"executor for tasks: {tasks} already exist.")
```

Comment:
What if two executors share some tasks but not all? Or if they have the same tasks, but listed in different order.

Reply:
enhanced the common task checking for adding the executors.